src/compute/src/row_spine.rs

```diff
@@ -326,7 +326,10 @@ mod container {
 
     use mz_repr::fixed_length::ToDatumIter;
     impl<'long> ToDatumIter for DatumSeq<'long> {
-        type DatumIter<'short> = DatumSeq<'short> where Self: 'short;
+        type DatumIter<'short>
+            = DatumSeq<'short>
+        where
+            Self: 'short;
```

Comment:
Lol, how does `rustfmt` find this better

Reply:
Lol I know, I was quite surprised with this one 😂